### PR TITLE
Our team cards component

### PR DESCRIPTION
**Pull request by jennetturkkan**
* added each member card that includes image, github and linkedIn links, name and title
* added translation info
* added snapshot test

Screen Shot 2020-11-11 at 13 05 34

### CODE REVIEW

**Review by hiba-machfej — APPROVED** [review]
looks good to me

File: src/components/OurTeamCards/index.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                src={memberImagesMap[member.portraitImg] || defaultImg}
+                alt={member.name}
+                className="portraitImg"
+              />
+              <div>
+                {member.gitHubLink && (
+                  <a
+                    href={member.gitHubLink}
+                    target="_blank"
+                    rel="noopener noreferrer"
```
Comment (mohammad-jundi) [inline src/components/OurTeamCards/index.jsx:46]: I couldn't understand what noopener noreferrer, can you use something else.

**Review by mohammad-jundi — COMMENTED** [review]
in general, it looks good and clean to me

File: src/components/OurTeamCards/index.jsx (hunk 1 above)
Comment (jennetturkkan) [reply src/components/OurTeamCards/index.jsx:46]: in React you cannot use `target="_blank"` without `rel="noopener noreferrer"` for security reasons

**Review by mohammad-jundi — APPROVED** [review]
looks good, I like the pictures naming

File: public/locales/en/translation.json [hunk 2]
```diff
@@ -127,6 +127,50 @@
...
-    "bodyText": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor i reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+    "bodyText": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
+    "subHeader": "Our team",
+    "ourTeamInfo": [
+      {
+        "name": "Hiba Machfej",
+        "title": "Junior Engineer",
+        "gitHubLink": "https://github.com/hiba-machfej",
+        "linkedInLink": "https://www.linkedin.com/in/hiba-machfej-09400189/",
```
Comment (louisrli) [inline public/locales/en/translation.json]: linkedInUrl

Comment (louisrli) [reply public/locales/en/translation.json]: gitHubUrl

Comment (louisrli) [reply public/locales/en/translation.json]: link can mean a lot of things in coding

**Review by louisrli — APPROVED** [review]
louisLiFrowning.jpg

**Comment by jennetturkkan** [comment]
> Frowning

my PR made you frown? :D

File: public/locales/en/translation.json (hunk 2 above)
Comment (jennetturkkan) [reply public/locales/en/translation.json]: fixed

**Comment by jennetturkkan** [comment]
@Ammar-64 can you please merge?

**Comment by Ammar-64** [comment]
> @Ammar-64 can you please merge?

No

**Comment by jennetturkkan** [comment]
> > @Ammar-64 can you please merge?
> 
> No

why? :(

**Pull request merged**